readme.md

```diff
@@ -42,6 +42,7 @@ We use Twitter a lot and notice many dumb annoyances we'd like to fix. So here b
 - Uses your personal color theme on all profiles.
 - Hides the header image on profile pages.
 - [Shows alternative image text below images when available.](https://user-images.githubusercontent.com/170270/40556400-b46c292c-6076-11e8-8241-f5c4e1a7a161.png)
+- Anonymizes tweets by replacing real usernames with "John Doe" or "Jane Doe"
```

Comment:
```suggestion
- Anonymizes tweets by replacing real usernames and names with `John Doe` or `Jane Doe`.
```

Reply:
This made be realize, it shouldn't replace usernames with `John Doe`. It should rather replace those with `@johndoe` or `@janedoe`. Right?